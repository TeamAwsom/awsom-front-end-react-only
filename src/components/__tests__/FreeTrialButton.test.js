import React from 'react';
import ReactDOM from 'react-dom';
import FreeTrialButton from '../FreeTrialButton';
import renderer from 'react-test-renderer';

describe('Free Trial Button', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<FreeTrialButton />).toJSON();

    console.log(tree);
    expect(tree).toMatchSnapshot();
  });
});
