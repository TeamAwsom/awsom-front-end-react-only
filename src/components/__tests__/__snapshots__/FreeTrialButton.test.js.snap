// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Free Trial Button renders as expected 1`] = `
<button
  className="btn freetrial_btn"
  type="button"
>
  FREE TRIAL
</button>
`;
